app/views/events/index.html.erb

```diff
@@ -16,15 +16,15 @@
           <td><%= event.name %></td>
           <td><%= event.duration %></td>
           <td><%= event.started_at %></td>
-          <td><%= link_to 'Show', event %></td>
+          <td><%= link_to '<%= t('.setting_updated')%>', event %></td>
```

Comment:
You cannot use `<%= %>` inside `<%= %>`. You should use pure `t('.setting_updated')` structure in whole this file.